Add prep for How our curriculum works/self educate (#646)
CodeYourFuture/curriculum: common-content/en/module/how-our-curriculum-works/active-learning/index.md

+++
title="🎬 Active learning"
headless="true"
time= 30
[objectives]
    1="Define active learning"
+++

> Active learning is at the heart of Code Your Future's educational philosophy. It is about encouraging learners to build, explore and discuss new concepts instead of being passive recipients of knowledge.

Reviewer: ```suggestion
Active learning is at the heart of our educational philosophy. It is about encouraging learners to build, explore and discuss new concepts instead of being passive recipients of knowledge.
```

Reviewer: If you want to use this with MigraCode let's drop CYF where pos
